edit-post/assets/stylesheets/_animations.scss

```diff
@@ -57,3 +57,17 @@
 @mixin animate_rotation($speed: 1s) {
 	animation: rotation $speed infinite linear;
 }
+
+@keyframes modal-appear {
+	from {
+		margin-top: $grid-size * 4;
```

Comment:
Animating the margin is potentially not as performant as translate, but we can't use translate as we use that for responsive positioning at responsive breakpoints. @kjellr do you have any insights as to whether we can make this more performant? Can we use `will-change: transform;` yet? Or are we still using `translateZ(0)` for this?

Reply:
Hmm, I see what you mean — we're sort of backed into animating the margin here because of the other `transform` properties we're using. We could theoretically use translate if we declared separate, super-specific animations for each of the two breakpoints:

```
@keyframes modal-appear-small {
	from {
		transform: translate(-50%, 10%);
	}
	to {
		transform: translate(-50%, 0);
	}
}

@mixin modal_appear_small() {
	animation: modal-appear-small 0.1s ease-out;
	animation-fill-mode: forwards;
}

@keyframes modal-appear-medium {
	from {
		transform: translate(-50%, -20%);
	}
	to {
		transform: translate(-50%, -30%);
	}
}

@mixin modal_appear_medium() {
	animation: modal-appear-medium 0.1s ease-out;
	animation-fill-mode: forwards;
}
```

And then declarations like these in `/modal/style.css`:

```
@include break-small() {
	...

	// Animate appearance.
	@include modal_appear_small();
}
@include break-medium () {
	...

	// Animate appearance.
	@include modal_appear_medium();
}
```

... but that feels like a bit of a hack (and a longwinded one at that), and I'm not 100% sure that it gains us a lot in terms of performance. So maybe we just stick to animating the margin?

Regarding `will-change`: I'm not totally familiar with that property, but I'm getting mixed signals about its usefulness [when reading the MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change). Browser support aside, they list a lot of caveats — based on what I'm reading there, I'm not sure an animation this small actually merits the use of it.